tests/unit/lms/resources/_js_config_test.py

```diff
@@ -159,3 +168,15 @@ def context():
         h_user=HUser("TEST_AUTHORITY", "example_username"),
         h_groupid="example_groupid",
     )
+
+
+@pytest.fixture
+def no_lti_user(pyramid_request):
+    """Modify the pyramid_request fixture so that request.lti_user is None."""
+    pyramid_request.lti_user = None
+
+
+@pytest.fixture
+def provisioning_disabled(context):
+    """Modify context so that context.provisioning_enabled is False."""
+    context.provisioning_enabled = False
```

Comment:
These fixtures deliberately don't return anything. They're not meant to be used as parameters of test methods, but rather in `@pytest.mark.usefixtures` decorators. They're not test objects as such, they're little bits of test set up wrapped in fixtures.